tests/unit/test_salesforce.py

```diff
@@ -0,0 +1,101 @@
+import pandas as pd
+import pytest
+
+from viadot.sources import Salesforce
+
+TABLE_TO_DOWNLOAD = "Account"
+TABLE_TO_UPSERT = "Contact"
+TEST_LAST_NAME = "prefect-viadot-test"
+ID_TO_UPSERT = "0035E00001YGWK3QAP"
+
+
+@pytest.fixture(scope="session")
+def salesforce():
+    s = Salesforce(config_key="sales_force_dev")
```

Comment:
Salesforce is written together, so should be `salesforce_dev`

Reply:
[🔐 Updated config_key name after git revert](https://github.com/dyvenia/viadot/pull/629/commits/a8fa33590c9a266a5ac5f9099b6ffed78e700b08)